docs/fundamentals/ux-text-style-guide/main-menu-functions.md

```
@@ -40,47 +75,110 @@
 
-| Dos                | Don'ts         |
-| ------------------ | -------------- |
-| Application status |                |
-| KPI settings       | KPI's settings |
-| KPI settings       | KPIS settings  |
-| KPIs               | KPI's          |
+<!-- markdownlint-disable MD033 -->
+<div class="dos-and-donts" markdown>
+<div class="dos" markdown>
+
+#### Do's
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)